integration/mediation-tests/tests-mediator-1/src/test/java/org/wso2/carbon/esb/mediator/test/db/dbreport/DBReportMediatorTestCase.java

```diff
@@ -29,49 +33,30 @@
 import org.wso2.carbon.automation.test.utils.dbutils.H2DataBaseManager;
 import org.wso2.esb.integration.common.utils.ESBIntegrationTest;
 
-import java.io.BufferedInputStream;
-import java.io.File;
-import java.io.FileInputStream;
-import java.io.IOException;
-import java.sql.ResultSet;
-import java.sql.SQLException;
-import java.sql.Statement;
-import java.util.Random;
-import javax.xml.namespace.QName;
-import javax.xml.stream.XMLInputFactory;
-import javax.xml.stream.XMLStreamException;
-import javax.xml.stream.XMLStreamReader;
-
 import static org.testng.Assert.assertEquals;
 
 public class DBReportMediatorTestCase extends ESBIntegrationTest {
 
     private H2DataBaseManager h2DataBaseManager;
-    private String JDBC_URL;
-    private String DB_USER;
-    private String DB_PASSWORD;
-    private String JDBC_DRIVER;
 
     @BeforeClass(alwaysRun = true)
     public void setEnvironment() throws Exception {
         super.init();
         AutomationContext automationContext = new AutomationContext();
-        DB_PASSWORD = automationContext.getConfigurationValue(XPathConstants.DATA_SOURCE_DB_PASSWORD);
-        JDBC_URL = automationContext.getConfigurationValue(XPathConstants.DATA_SOURCE_URL);
-        DB_USER = automationContext.getConfigurationValue(XPathConstants.DATA_SOURCE_DB_USER_NAME);
-        JDBC_DRIVER = automationContext.getConfigurationValue(XPathConstants.DATA_SOURCE_DRIVER_CLASS_NAME);
+        String DB_USER = "wso2carbon";
+        String DB_PASSWORD = "wso2carbon";
+        String JDBC_URL = automationContext.getConfigurationValue(XPathConstants.DATA_SOURCE_URL);
         String databaseName =
-                System.getProperty("basedir") + File.separator + "target" + File.separator + "testdb_dbreport"
-                        + new Random().nextInt();
+                System.getProperty("carbon.home") + File.separator + "databases" + File.separator + "testdb_dbreport";
```

Comment:
Shall we create this outside the pack and do the tests.

Reply:
Is there a specific reason that requires us to do so?

P.S. This is similar to #371.

Reply:
Here we are creating the databases directory inside product and adding these h2 DB. It is not a best practice to create a custom folder inside the product pack and use it for test case. Though this is a test case, since it is particular to only this test cases, while debugging other test cases or etc. it would lead to un-necessary confusions. Since we have the option to create in target ( as it was done previously ) why don't we do the same here. 
It would be better if we can modify https://github.com/wso2/micro-integrator/pull/371 too. WDYT.

Reply:
Please check 40361bb.